src/olympia/reviewers/tests/test_views.py

```diff
@@ -6808,11 +6824,15 @@ def test_disable_addon_does_not_exist(self):
         self.grant_permission(self.user, 'Reviews:Admin')
         self.client.login_api(self.user)
         self.disable_url = reverse_ns(
-            'reviewers-addon-enable', kwargs={'pk': self.addon.pk + 42}
```

Comment:
I take it this was a bug in the existing version of the test (pointing to `enable` instead of `disable`)?

Reply:
Yeah, copy/paste error in the previous version of the test